client/me/developer/style.scss

```
@@ -84,3 +84,106 @@
 	}
 }
 
+.developer-survey-notice {
+	position: fixed;
+	left: 0;
+	top: 0;
+	height: 100%;
+	width: 100%;
+	z-index: 1000;
+
+	.developer-survey-notice__backdrop {
+		background: var(--studio-black);
+		opacity: 0.2;
+		position: absolute;
+		left: 0;
+		top: 0;
+		width: 100%;
+		height: 100%;
+		cursor: default;
+	}
+	.developer-survey-notice__popup {
```

```suggestion
	}

	.developer-survey-notice__popup {
```

Nit. Missing new line. There also a few other places in this file that's missing a new line.